frontend/src/routes/Infrastructure/InfraEnvironments/CreateInfraEnv.tsx

```diff
@@ -139,6 +139,7 @@ const CreateInfraEnv: React.FC = () => {
                 creationStatus: creationStatus?.status,
                 creationMsg: creationStatus?.messages,
               }}
+              localHubName={localHubName}
```

Comment:
I don't think this will be used during creation of an infrastructure environment. Is it a required prop?